src/main/kotlin/lotto/LottoNumber.kt

```diff
@@ -0,0 +1,16 @@
+package lotto
+
+class LottoNumber {
```

Comment:
```suggestion
class LottoNumber(
    val value: Int
) {
```
LottoNumber 클래스도 반드시 숫자를 가집니다.
그렇다면 캡슐화하도록 개선해보면 어떨까요?

Reply:
캡슐화 진행하였습니다!